python/lsst/ts/FiberSpectrograph/avsSimulator.py

```diff
@@ -59,11 +61,14 @@ def mock_getList(a_listSize, a_pRequiredSize, a_pList):
             return self.n_devices
         config['return_value.AVS_GetList.side_effect'] = mock_getList
 
-        self.n_pixels = 3141
-        self.temperature_setpoint = -273.0
-        # thermistor voltage is converted to temperature via a polynomial
-        self.tec_coefficients = np.array((12, 34, 56, 78, 90), dtype=np.float32)
-        self.tec_voltage = 6.4
+        # Have the number of pixels, and temperature values match the real
+        # device, so that users aren't confused by simulation telemetry.
+        self.n_pixels = 2048
+        self.temperature_setpoint = 5
+        # thermistor voltage is converted to temperature via a polynomial:
+        # these coefficients should result in a temperature of 5.0
+        self.tec_coefficients = np.array((5, 0, 0, 0., 0), dtype=np.float32)
+        self.tec_voltage = 1
```

Comment:
Note that `temperature` is not part of the mock itself, but rather part of the internal simulator state. The mock itself (in this case `mock_getParameter` and `mock_getAnalogIn`, which replace `libavs.AVS_GetParameter` and `libavs.AVS_GetAnalogIn` respectively) doesn't know about the temperature, only the voltage and coefficients. I think I prefer keeping the temperature as "internal" state in the simulator, since there are two tests that check the temperature: one in the csc tests and one in the device controller tests.